src/screens/LoginPage/LoginPage.tsx

```diff
@@ -57,6 +64,53 @@ function loginPage(): JSX.Element {
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [showConfirmPassword, setShowConfirmPassword] =
     useState<boolean>(false);
+  const [showAlert, setShowAlert] = useState<PasswordValidation>({
+    lowercaseChar: true,
+    uppercaseChar: true,
+    numericValue: true,
+    specialChar: true,
+  });
+
+  const passwordValidationRegExp = {
+    lowercaseCharRegExp: new RegExp('[a-z]'),
+    uppercasaeCharRegExp: new RegExp('[A-Z]'),
+    numericalValueRegExp: new RegExp('\\d'),
+    specialCharRegExp: new RegExp('[!@#$%^&*()_+{}\\[\\]:;<>,.?~\\\\/-]'),
+  };
+  const handleLowercasePassCheck = (pass: string): void => {
+    const success = passwordValidationRegExp.lowercaseCharRegExp.test(pass);
+    if (success) {
+      setShowAlert((prevAlert) => ({ ...prevAlert, lowercaseChar: false }));
+    } else {
+      setShowAlert((prevAlert) => ({ ...prevAlert, lowercaseChar: true }));
+    }
+  };
+
+  const handleUppercasePassCheck = (pass: string): void => {
+    const success = passwordValidationRegExp.uppercasaeCharRegExp.test(pass);
+    if (success) {
+      setShowAlert((prevAlert) => ({ ...prevAlert, uppercaseChar: false }));
+    } else {
+      setShowAlert((prevAlert) => ({ ...prevAlert, uppercaseChar: true }));
+    }
+  };
+  const handleNumericalValuePassCheck = (pass: string): void => {
+    const success = passwordValidationRegExp.numericalValueRegExp.test(pass);
+    if (success) {
+      setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: false }));
+    } else {
+      setShowAlert((prevAlert) => ({ ...prevAlert, numericValue: true }));
+    }
+  };
+  const handleSpecialCharPassCheck = (pass: string): void => {
+    const success = passwordValidationRegExp.specialCharRegExp.test(pass);
```

Comment:
You can directly add 
```ts
if (passwordValidationRegExp.specialCharRegExp.test(pass)){
} 
```

to your code, there is no need of declaring a variable, since you aren't using it multiple times

Reply:
But I am using the `handleSpecialCharPassCheck` method in `onChange` event on `password` input so we will need the handlers right?

Reply:
<img width="657" alt="image" src="https://github.com/PalisadoesFoundation/talawa-admin/assets/116940083/96b8d7ba-7f92-412b-9391-6d8254a3bdce">

Reply:
@MahendraDani I was talking about the success variable in your methods, you are using it only once, so that is why its not required

Reply:
So that the text color of validation alerts change dynamically when they pass or fail